ichnaea/log.py

```diff
@@ -89,7 +89,8 @@ def configure_raven(transport=None, _client=None):
 
     dsn = settings("sentry_dsn")
     klass = DebugRavenClient if not dsn else RavenClient
-    release = version_info()["tag"]
+    info = version_info()
+    release = info.get("tag") or info.get("commit") or "unknown"
```

Comment:
Oh, whoops... this isn't right. This works with the `version.json` that's in the repo, but that doesn't match the one that's built by `.circleci/config.yml`.

Mmm... I'll fix all of them.